cp3pt0-deployment/setup_tenant.sh

```diff
@@ -270,6 +270,13 @@ function default_arguments() {
         else
             info "CommonService CRD exists but main CommonService CR does not exist, skipping defaulting from original CommonService CR\n"
         fi
+
+        # if CommonService CRD exists and subscription of common-service-operator exists, simple topology will be accepted
```

Comment:
Are we sure we want to add the code not relevant to LSR?

Reply:
it is merge from origin repo